test_scripts/Resumption/InteriorVehicleData/021_InteriorVD_resumption_in_3rd_ignition_cycle.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL resumes interior vehicle data in 3rd ignition cycle
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1)
+--
+-- Sequence:
+-- 1. Two IGN_OFF and IGN_ON cycles are performed
+-- 2. App starts registration with actual hashId after IGN_ON in 3rd ignition cycle
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1, default moduleId) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
b. SDL get a successful response for GetInteriorVehicleData request with isSubscribed=true from HMI
c. SDL respond RAI(success=true, result code = SUCCESS) to mobile app
d. update hashId after successful resumption

Reply:
Done